packages/collections/graphql/resolvers/DocumentProgress.js

```diff
@@ -0,0 +1,13 @@
+const Collection = require('../../lib/Collection')
+
+module.exports = {
+  max: item => {
+    const max = Collection.getItemMax(item)
```

Comment:
This reads odd to me.

If `max` is truthy, object with (overwritten) `id` is returned. Got it so far. If `max` is however falsy, it will return `max` - which the leaves it to be falsy e.g. `undefined`, `null`, `false`.

Is that intentional? Wouldn't it be safer to return `null`, in respect with schema, and since resolver is "last resort"?

Same consideration applies to sibling file `packages/collections/graphql/resolvers/MediaProgress.js`

Reply:
Good point, returning `null` is much cleaner.